### ISSUE TITLE
[FEATURE] Migrate the landing to Next.js

### ISSUE THREAD

**Issue by mkubdev** [WebXDAO/WebXDAO.github.io#151]
### Detailed description

We should migrate the landing page into nextjs since we will mostly use this framework for the org's projects.

## Todo
- Migrate the landing page to next.js framework
- Use TailwindUI
- Deploy to GH page : https://dev.to/jameswallis/deploying-a-next-js-app-to-github-pages-24pn

### Possible implementation

Use next.js + tailwind

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by krishguptadev** [comment]
Oh, so I should remove the stuff we've done in Astro?

**Comment by shuklaritvik06** [comment]
> Oh, so I should remove the stuff we've done in Astro?

The new version of the website is in Astro right?

**Comment by mkubdev** [comment]
> Oh, so I should remove the stuff we've done in Astro?

We can keep this version with Astro, but if we have a new landing design, we should migrate to nextjs. We can save dev.to articles logic request.

**Comment by vinzvinci** [comment]
Nice @mkubdev 🎉

**Comment by krishguptadev** [comment]
> > Oh, so I should remove the stuff we've done in Astro?
> 
> We can keep this version with Astro till we make a new PR with the nextjs. We can save dev.to articles logic request.

or since we haven't done much in Astro and converting Astro to react isn't pretty hard, we can do nextjs

**Comment by mkubdev** [comment]
> or since we haven't done much in Astro and converting Astro to react isn't pretty hard, we can do nextjs

@krishguptadev Yeah absolutly right!

**Comment by AbhinavMV** [comment]
is someone working on this? 
If not i would be happy to take this issue 😄

**Comment by mkubdev** [comment]
Hi @AbhinavMV ! Sure you can make a PR with a nextjs stack! :)

Don't bother to adjust the design, since we are rebranding, we will update the landing page as well.

🌱

**Comment by AbhinavMV** [comment]
> 

So i just need to delete all the code and create a initialize with nextjs+tailwind 
I don't need to convert any code to nextjs ? 🤔

**Comment by mkubdev** [comment]
@AbhinavMV Ok, i see. Maybe you can take the actual landing, and making components of Hero, Section, Footer, etc ?

**Comment by AbhinavMV** [comment]
> @AbhinavMV Ok, i see. Maybe you can take the actual landing, and making components of Hero, Section, Footer, etc ? We will see after to match Github Action build with nextjs. Or maybe you know how to manage it? :)

cool i will give it try 👍
Once page is ready i will look into github actions

**Comment by mkubdev** [comment]
I close, solved with #162 , Thanks again Ser @AbhinavMV 😎
